### PR TITLE
Index: Add SerenityOS to the table

### PR DESCRIPTION
**Pull request by bgianfo**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -83,6 +83,10 @@ <h2>Join the Club</h2>
           <td><a href="https://github.com/natalie-lang/natalie">Natalie</a></td>
           <td>a Ruby compiler, for fun</td>
         </tr>
+        <tr>
+          <td><a href="https://github.com/SerenityOS/serenity">SerenityOS</a></td>
+          <td>a operating system from scratch, for fun</td>
```
Comment (ADKaster) [inline index.html:88]: @seven1m Where does the restriction 

> The first word must be a lowercase "a".

come from? Perhaps that consumer should be updated to recognize `an` as well, for better grammaticallness :)

Comment (seven1m) [reply index.html:88]: Yeah, that should be a/an. 😄

Comment (bgianfo) [reply index.html]: I fixed it, and update the comment and PR template directions. Thanks.

Comment (bgianfo) [reply index.html]: just kidding... looks like Tim beat me to it.

**Comment by seven1m** [comment]
As Andreas said in SerenityOS/serenity#16201, Serenity is now bigger than the "just for fun" mindset.

Hopefully there will be a bunch of smaller, more obscure projects who need this type of exposure.

Thank you friends! 🤗

**Pull request closed without merging**